api/v1beta1/loadbalancerset_types.go

```diff
@@ -4,6 +4,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// CreationTimeoutAnnotation can be set to override the defaut duration of
+	// 10 minutes, after which a new LoadBalancerMachine is deleted (by the LBM)
+	// if it reports no conditions yet.
+	CreationTimeoutAnnotation = "yawol.stackit.cloud/creationTimeout"
```

Comment:
I'm a bit unsure about the naming here, feel free to suggest something else

Reply:
I don't have a better suggestion for the annotation key itself. Sounds ok to me. At least, the case is consistent with other annotations in the project.

I would call the constant `LoadBalancerMachineCreationTimeoutAnnotation` to indicate the object kind where it can be set (similar to other projects in `loadbalancer_types.go`.